<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Surah</string>
     </property>
     <widget class="QComboBox" name="surahComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">combobox-popup: 0;</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ayahs Range</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>From</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>To</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditTo">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditFrom">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Reciter</string>
     </property>
     <widget class="QComboBox" name="reciterComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">combobox-popup: 0;</string>
      </property>
      <item>
       <property name="text">
        <string>Maher Al-Muaiqly</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Abu Bakr Ash-Shaatree</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Destination</string>
     </property>
     <widget class="QPushButton" name="browseBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditFilePath">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="downloadBtn">
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
